src/test/gcp/cloudfunctionsv2.spec.ts

```diff
@@ -460,8 +462,96 @@ describe("cloudfunctionsv2", () => {
       };
 
       expect(
-        cloudfunctionsv2.functionFromSpec(complexFunction, CLOUD_FUNCTION_V2_SOURCE)
+        await cloudfunctionsv2.functionFromSpec(complexFunction, CLOUD_FUNCTION_V2_SOURCE)
+      ).to.deep.equal(complexGcfFunction);
```

Comment:
nit* This is identical to

```
await expect(asuncFn()).to.eventually.be.deep.equal(expected);
```

Reply:
Changed the method and removed this code change